File: app/src/main/java/com/BlizzardArmory/starcraft/Profile/Criterium.java

package com.BlizzardArmory.starcraft.Profile;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Criterium {

    @SerializedName("criterionId")
    @Expose
    private String criterionId;

    public String getCriterionId() {
        return criterionId;
    }

    public void setCriterionId(String criterionId) {
        this.criterionId = criterionId;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/starcraft/profile/Criterium.kt
package com.BlizzardArmory.model.starcraft.profile

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Criterium.
 */
@Keep
data class Criterium(

    @SerializedName("criterionId")
    var criterionId: String,

    @SerializedName("earned")
    var earned: Earned

)